openapi-generator/src/main/kotlin/org/jellyfin/openapi/builder/openapi/OpenApiModelBuilder.kt

```diff
@@ -59,6 +59,7 @@ class OpenApiModelBuilder(
 						deprecated = it.deprecated,
 					)
 				}.toSet()
+
```

Comment:
```suggestion
```

New IntelliJ strikes, might want to add `ij_kotlin_line_break_after_multiline_when_entry = false` to the `.editorconfig`.